merge_fixtures_test.go

```diff
@@ -34,6 +34,10 @@ func (f MergeTestFixture) CheckSuccess(t *testing.T) {
 	}
 	actual := mustMergeSchemas(t, schemas...)
 
+	if actual == nil {
+		return
+	}
+
 	// If resulting Query type is empty, remove it from schema to avoid
 	// generating an invalid schema when formatting (empty Query type: `type Query {}`)
 	if actual.Query != nil && len(filterBuiltinFields(actual.Query.Fields)) == 0 {
```

Comment:
This is where the nil pointer dereference happens.

Reply:
`mustMergeSchemas` can return `nil`: https://github.com/movio/bramble/blob/70a8e42e99a0c150bf8367dd467270d7d86bf641/merge_fixtures_test.go#L93